enterprise/cmd/frontend/internal/dotcom/productsubscription/license_expiration.go

```diff
@@ -0,0 +1,81 @@
+package productsubscription
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/inconshreveable/log15"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/db"
+	"github.com/sourcegraph/sourcegraph/enterprise/cmd/frontend/internal/licensing"
+	"github.com/sourcegraph/sourcegraph/internal/slack"
+)
+
+var SourcegraphOrgWebhookURL = env.Get("SLACK_LICENSE_EXPIRATION_BOT_HOOK", "", "Webhook for announcing license expirations to a Slack channel for Sourcegraph.com.")
```

Comment:
Looks like this is only used by this package internally?
```suggestion
var sourcegraphOrgWebhookURL = env.Get("SLACK_LICENSE_EXPIRATION_BOT_HOOK", "", "Webhook for announcing license expirations to a Slack channel for Sourcegraph.com.")
```

Reply:
Replaced by a site config option